apps/studio-next/src/components/Modals/ImportUUIDModal.tsx

```diff
@@ -0,0 +1,57 @@
+import { useState } from 'react';
+import toast from 'react-hot-toast';
+import { create } from '@ebay/nice-modal-react';
```

Comment:
I see that we've used this library without an ADR, It's fine.
Could we just add an ADR for it, to understand the reasoning behind it.

Reply:
Sure, let me check in which commit it was introduced initially.

Reply:
@Amzani it was introduced in this PR https://github.com/asyncapi/studio/pull/518.